Xamarin.Essentials/Calendar/Calendar.android.cs

```diff
@@ -128,38 +133,141 @@ static async Task<IEvent> PlatformGetEventByIdAsync(string eventId)
-                        Status = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Status)),
-                    };
-                }
-                throw new NullReferenceException($"[Android]: No Event found for event Id {eventId}");
+                    Id = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Id)),
+                    CalendarId = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.CalendarId)),
+                    Title = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Title)),
+                    Description = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Description)),
+                    Location = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.EventLocation)),
+                    AllDay = cur.GetInt(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.AllDay)) == 1,
+                    Start = cur.GetLong(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Dtstart)),
+                    End = cur.GetLong(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Dtend)),
+                    HasAlarm = cur.GetInt(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.HasAlarm)) == 1,
+                    HasAttendees = cur.GetInt(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.HasAttendeeData)) == 1,
+                    HasExtendedProperties = cur.GetInt(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.HasExtendedProperties)) == 1,
+                    Status = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Status)),
+                    Attendees = GetAttendeesForEvent(eventId),
+                    RecurrancePattern = !string.IsNullOrEmpty(rRule) ? GetRecurranceRuleForEvent(rRule) : null
+                };
             }
             catch (NullReferenceException)
             {
                 throw new NullReferenceException($"[Android]: No Event found for event Id {eventId}");
             }
         }
+
+        static IReadOnlyList<IAttendee> GetAttendeesForEvent(string eventId)
+        {
+            var attendeesUri = CalendarContract.Attendees.ContentUri;
+            var attendeesProjection = new List<string>
+            {
+                CalendarContract.Attendees.InterfaceConsts.EventId,
+                CalendarContract.Attendees.InterfaceConsts.AttendeeEmail,
+                CalendarContract.Attendees.InterfaceConsts.AttendeeName
+            };
+            var attendeeSpecificAttendees = $"{CalendarContract.Attendees.InterfaceConsts.EventId}={eventId}";
+            var cur = Platform.AppContext.ApplicationContext.ContentResolver.Query(attendeesUri, attendeesProjection.ToArray(), attendeeSpecificAttendees, null, null);
```

Comment:
I can't tell but should we be disposing of the `cur`? Ignore this if not

Reply:
Not 100% but can't do any harm have added it accordingly